recipes/atlas-schema/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "atlas-schema" %}
+{% set version = "0.1.0" %}
+{% set python_min = "3.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/atlas_schema-{{ version }}.tar.gz
+  sha256: 50f8fd29e958f00ff5fe8b0f2235e281952e5391e893bb38e61c2c50751b7766
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv # --no-deps --no-build-isolation
```

```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

These are already enforced by the build tool (e.g. `conda-build`, `rattler-build`).

---

this came from grayskull, so it wasn't obvious if i can drop it, given the conda documentation recommended grayskull.